src/pages/demo/bannerAlert/index.tsx

```diff
@@ -36,22 +35,20 @@ export async function getStaticProps(
     'field_banner_alert_vamcs',
     'field_banner_alert_vamcs.field_office',
   ])
-  params.addFilter('field_banner_alert_vamcs', null, 'IS NOT NULL')
```

Comment:
How is this object being returned now?

Reply:
for some reason when we leave in line 39 and generate all 1500 banners we get into a call-stack loop partway though, I messed with this for about an hour on Friday and finally resolved the build it by removing this line altogether. The include on line 36 seems to cover the object we need back but adding the null check causes problems and I don't really understand why